src/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -160,5 +165,8 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
+  <Target Name="CopyLatestJsonFile" AfterTargets="Build" >
```

Comment:
I don't think you want to do this.  That's assuming that the tool runtime is targeting the same framework that you're building this app for, not to mention it's clashing with the file you have above.